lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -1609,6 +1612,11 @@ static uint32_t LookupSymbolInModule(CommandInterpreter &interpreter,
   }
 
   if (num_matches > 0) {
+    llvm::StringRef ansi_prefix =
+        interpreter.GetDebugger().GetRegexMatchAnsiPrefix();
+    llvm::StringRef ansi_suffix =
+        interpreter.GetDebugger().GetRegexMatchAnsiSuffix();
+    Information info(name, ansi_prefix, ansi_suffix);
```

Comment:
Construct this closer to the place of use, also inline the `GetRegex...` calls into it like `info(...GetRegexMatch...())`.

Reply:
I'm unsure if the modification made in last commit addresses this correctly.